evolution/engine.py

```
@@ -823,17 +864,17 @@ def _emit_event(self, event_type: str, data: Dict[str, Any]):
                     callback(data)
                 except Exception as e:
                     logger.error(f"Event listener error: {e}")
-        
+
         # New async event bus with backpressure
         asyncio.create_task(self._emit_async_event(event_type, data))
```

Ruff (0.14.7)

Store a reference to the return value of `asyncio.create_task`

(RUF006)